SofaKernel/modules/SofaSimulationCommon/src/SofaSimulationCommon/xml/ObjectElement.cpp

```diff
@@ -91,6 +91,13 @@ bool ObjectElement::initNode()
         msg_error(ctx) << tmp.str() ;
         return false;
     }
+    auto& objName = *obj->name.beginEdit();
```

Comment:
It is better to not use beginEdit/endEdit anymore. Using Read/Write accessor is better.

Reply:
I agree, but accessors do not support `Data<std::string>`. Your suggestion cannot be applied.